src/components/buttons/secondary-icon-button/README.md

```diff
@@ -31,6 +31,11 @@ also pass a label for accessibility reasons.
 | `icon`       | `node`   |    ✅    | -                           | -        | An `Icon` component                                                                    |
 | `isDisabled` | `bool`   |    -     | -                           | `false`  | Tells when the button should present a disabled state                                  |
 | `onClick`    | `func`   |    ✅    | -                           | -        | What the button will trigger when clicked                                              |
+| `color`      | `oneOf`  |    -     | `solid`, `primary`          | `solid`  | Sets the color of the icon                                                             |
+
+#### Note
+
+The size of the button should be adjusted directly on the passed `Icon` component.
```

Comment:
😄

Reply:
Nitpick: a short code snippet as an example would make it even better 😉

Reply:
Sounds good 👍